mod_pywebsocket/websocket_server.py

```diff
@@ -85,11 +85,18 @@ def __init__(self, options):
         if necessary.
         """
 
+        try:
```

Comment:
This works as anticipated, but maybe we should consider using hasattr() in this case as well?

Reply:
I found this article telling me not to use hasattr(): https://hynek.me/articles/hasattr/.

But it also told me how to use getattr() instead. So I did that.